autoload/yankitute.vim

```diff
@@ -75,3 +77,11 @@ function! s:eval(results, replace) abort
     call add(a:results, eval(a:replace[2:]))
     return submatch(0)
 endfunction
+
+function! s:expand_returns(lst) abort
+  let arr = []
+  let pat = '\v%(%(\\\\)*\\)@<!%(\\r)'
```

Comment:
Is this actually working? This would split strings containing `'\r'` - as in backslash followed by `r`, and not carriage return. This looks like a command for splitting the source patterns - but you are applying it on the result lines?

BTW - in the result lines' case - wouldn't `'\r'` suffice?